test/Identity.FunctionalTests/LoginTests.cs

```diff
@@ -65,8 +87,14 @@ public LoginTests(ServerFactory<TStartup, TContext> serverFactory)
         public async Task CanLogInWithTwoFactorAuthentication_WithGlobalAuthorizeFilter()
         {
             // Arrange
-            var client = ServerFactory.CreateClient();
-            var newClient = ServerFactory.CreateClient();
+            void ConfigureTestServices(IServiceCollection services) =>
```

Comment:
Looks like this test was accidentally refactored incorrectly here https://github.com/aspnet/Identity/commit/f51af820a5ccf2268b785c62140b29bbb5a24e83#diff-a17189606cd0186d845b9befa0cfa021R65